django-stubs/http/cookie.pyi

```diff
@@ -1,5 +1,3 @@
-from typing import Any
-
-SimpleCookie: Any
+from http.cookies import SimpleCookie as SimpleCookie
```

Comment:
Should we keep this or simply remove it?

Reply:
Let's keep it as long as Django has this alias.